File: Angular/SD.IdentitySystem.Client/src/models/role.ts
import {ModelBase} from "sd-infrastructure";
import {InfoSystem} from "./info-system";

/*角色*/
export interface Role extends ModelBase {

    /*信息系统编号*/
    infoSystemNo: string;

    /*描述*/
    description: string | null;

    /*导航属性 - 信息系统*/
    infoSystemInfo: InfoSystem | null;
}

File: Websites/SD.IdentitySystem.Presentation/Models/Role.cs
using SD.Infrastructure.PresentationBase;

namespace SD.IdentitySystem.Presentation.Models
{
    /// <summary>
    /// 角色
    /// </summary>
    public class Role : ModelBase
    {
        #region 角色描述 —— string Description
        /// <summary>
        /// 角色描述
        /// </summary>
        public string Description { get; set; }
        #endregion


        //Others

        #region 信息系统编号 —— string InfoSystemNo
        /// <summary>
        /// 信息系统编号
        /// </summary>
        public string InfoSystemNo { get; set; }
        #endregion

        #region 信息系统名称 —— string InfoSystemName
        /// <summary>
        /// 信息系统名称
        /// </summary>
        public string InfoSystemName { get; set; }
        #endregion
    }
}
